src/extension.ts

```diff
@@ -94,7 +95,11 @@ export async function activate(context: ExtensionContext) {
     };
     const testRunProfile = ctrl.createRunProfile('Run Tests', TestRunProfileKind.Run, runHandler, true, undefined, true);
     if (extensions.getExtension('xdebug.php-debug') !== undefined) {
-        ctrl.createRunProfile('Debug Tests', TestRunProfileKind.Debug, runHandler, false, undefined, false);
+        ctrl.createRunProfile('Debug Tests', TestRunProfileKind.Debug, runHandler, true, undefined, false);
+    }
+    const coverageProfile = ctrl.createRunProfile('Run with Coverage', TestRunProfileKind.Coverage, runHandler, true, undefined, false); // TODO Continuous
```

Comment:
I think line 100 - 102 should be moved to line 98

Reply:
Well no. Coverage depends on Xdebug PHP extension being installed, whereas debugging depends on Xdebug PHP extension _AND_ xdebug.php-debug vscode extension. We could add a test to check if Xdebug is installed in the currently selected PHP.

But I'm currently not doing this even in the debug vscode extension, so...